### PR TITLE
Few errors

### PR DESCRIPTION
**Pull request by Gasjki**
As I previously said, I had some errors when creating a new user. Anyway, I had a plan to extend your script, but until that moment, let's make this work for everyone. 👍

### CODE REVIEW

File: Security/Authentication/Token/SteamUserToken.php [hunk 1]
```diff
@@ -79,9 +79,9 @@ public function getUsername()
     }
 
     /**
-     * @param int $username
+     * @param string $username
      */
-    public function setUsername(int $username)
+    public function setUsername(string $username)
```
Comment (knojector) [inline Security/Authentication/Token/SteamUserToken.php]: The steamID64 is used as username and is an integer. Is there a reason for this change?

File: User/AbstractSteamUser.php [hunk 2]
```diff
@@ -13,7 +13,7 @@
     /**
      * @var string
      *
-     * @ORM\Column(type="bigint")
+     * @ORM\Column(type="string")
```
Comment (knojector) [inline User/AbstractSteamUser.php]: see above.

File: User/AbstractSteamUser.php [hunk 3]
```diff
@@ -76,7 +76,7 @@
     /**
      * @var int
      *
-     * @ORM\Column(type="bigint", nullable=true)
+     * @ORM\Column(type="string", nullable=true)
```
Comment (knojector) [inline User/AbstractSteamUser.php]: primaryClanId is also an integer.

File: User/AbstractSteamUser.php [hunk 4]
```diff
@@ -250,15 +250,15 @@ public function setPersonaState(? int $state)
...
+    public function getPrimaryClanId(): ? string
     {
         return $this->primaryClanId;
     }
 
     /**
      * {@inheritdoc}
      */
-    public function setPrimaryClanId(int $clanId)
+    public function setPrimaryClanId(?string $clanId)
```
Comment (knojector) [inline User/AbstractSteamUser.php]: Good idea, a user might not have a primary group selected.

**Comment by Gasjki** [comment]
First of all, after I installed your script all I did was to create a new user entity and to set configuration file. Then I tried to run `php bin/console d:s:u --force` and I had that error with database. And yes, I'm using MariaDB. 

I changed that because I had another error for those things like: `::setPrimaryClanId() was expected int, string given` so I decided to change that. That's because you get Steam ID and Primary Clan Id as strings from request (check it)

**Comment by Gasjki** [comment]
Any update ? :D

**Pull request closed without merging**